octobot_commons/enums.py

```diff
@@ -423,3 +423,13 @@ class UserInputTypes(enum.Enum):
     OBJECT = "object"
     OBJECT_ARRAY = "object_array"
     STRING_ARRAY = "string_array"
+
+
+class UserInputEditorOptionsTypes(enum.Enum):
+    GRID_COLUMNS = "grid_columns"
+    COLLAPSED = "collapsed"
```

Comment:
Will it be used somewhere?

Reply:
(talking about this)
![image](https://user-images.githubusercontent.com/9078616/226930492-928b00b3-9eec-4605-bd01-47f635e773a1.png)

Reply:
Thanks for these details!